tests/py/dynamo/conversion/harness.py

```diff
@@ -252,6 +251,40 @@ def run_test(
             check_dtype,
         )
 
+    def run_test_comparator(
+        self,
+        mod,
+        inputs,
+        expected_ops,
+        comparators: List[Tuple[Callable, List]],
+        precision=torch.float32,
+        output_dtypes=None,
+        use_dynamo_tracer=False,
+        enable_passes=False,
+    ):
```

Comment:
Does this only test the shapes of the results? If so, can the name be updated to `run_test_compare_shapes_only` or something similar?

Reply:
In this case it is checking the shape of the Tensors. But it is supposed to be a generic function where the callable can check other attribute of the Tensor. For example, in the case of `rand` and `randn`  we compare the data type as well. So I think the name should be `run_test_comparator` but you can let me know if you think otherwise.

Reply:
Ok that makes sense, I think the name is reasonable then. If possible, maybe `run_test_compare_tensor_attributes_only` just to delineate that the actual values in the tensors are ignored, but it is also okay as is

Reply:
Ok makes sense, I will rename it to the above then.